Writerside/topics/Shapes/Annotation-tools.md

```diff
@@ -0,0 +1,275 @@
+# Annotation tools
+<tldr>
+    <p>UI path: <ui-path><code>Draw view</code> | <code>Shape tool panel</code> | <code>Annotation tool set</code></ui-path></p>
+</tldr>
+
+<tip>
+<p>
+While drawing shapes, annotations, <code>point snap</code> options should be turned on to achieve the best accuracy.
+</p>
+</tip>
+<note>
+<p>
+Annotations do not belong to the Metric class; therefore, they cannot subscribe to a material. Annotations are free shapes, and their styling is always managed via the <code>style toolbar</code>.</p>
+</note>
+
+## Scale tool
+The <code>scale tool</code> is not used to draw a visual shape but is used to set a page scale.
+<procedure>
+<step>
+Click on the <code>scale tool</code> icon to activate the tool.
+</step>
+<step>
+Click on the first point on the <code>draw board</code> to start setting the scale.
+</step>
+<step>
+Move the mouse to the second point and click to set the scale.
```

Comment:
"Move the mouse to the second point and click to set the scale." ->"Move the mouse to the second point and click to get the model distance"